Tests/ApolloTests/GETTransformerTests.swift

```diff
@@ -90,7 +90,7 @@ class GETTransformerTests: XCTestCase {
   
   func testEncodingQueryWithNullDefaultParameter() {
     let operation = HeroNameQuery()
-    let body = requestCreator.requestBody(for: operation)
+    let body = requestCreator.requestBody(for: operation, sendOperationIdentifiers: false)
```

Comment:
Probably worth adding one test to this file where `sendOperationIdentifers` is `true` just to make sure that's actually including the operation identifier

Reply:
Well, the default implementation here was actually false. 
Maybe that is for another PR? 
I can open an issue to you can track it?

Reply:
Just not to add something that is not related to this PR and RequestCreator? 😁

Reply:
Yeah I'll add it in a follow-up PR - you don't need to open an issue for it, I'll leave this comment open